RMG/14Day/오픈채팅방.js

```diff
@@ -0,0 +1,50 @@
+function solution(record) {
+  // 사용자 정보를 저장할 객체
+  const users = {};
+  // 사용자 활동 기록을 저장할 배열
+  const orders = [];
```

Comment:
```suggestion
  const users = {}; // 사용자 정보를 저장할 객체
  const orders = []; // 사용자 활동 기록을 저장할 배열
```
주석을 다는 건 정말 좋습니다!

정말 사소한 거지만 주석이 코드 옆에 있으면 해당 코드와 주석이 시각적으로 더 가까워져, 주석이 설명하고 있는 대상을 한눈에 명확히 알 수 있어서 이렇게 빈 배열이나 빈 객체를 가진 (짧은)변수 같은 경우 코드 옆에 두는 게 가독성면으로 좀 더 좋습니다~